extensions/interactions/interaction-attributes-extractor.service.ts

```diff
@@ -25,6 +25,7 @@ import { InteractionCustomizationArgs } from
   'extensions/interactions/customization-args-defs';
 import { InteractionObjectFactory } from
   'domain/exploration/InteractionObjectFactory';
+import { InteractionId } from './rule-input-defs';
```

Comment:
Can we have the absolute import here? (If not, then add a new line above to separate relative imports from others.)

Reply:
Yes, we can have it. Thanks for pointing it. I'm adding a commit for this.